File: ignored-test/java-junit/InternalLoggerTests_Trace_ShouldThrowStackOverFlowExceptionWhenUsingNLogTraceListener.java
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Disabled;
import static org.junit.jupiter.api.Assertions.assertThrows;

public class InternalLoggerTests_Trace {

    @Test
    @Disabled("")
    public void shouldThrowStackOverflowExceptionWhenUsingNLogTraceListener() {
        setupTestConfiguration(NLogTraceListener.class, LogLevel.TRACE, true, null);
        assertThrows(StackOverflowError.class, () -> Trace.writeLine("StackOverFlowException"));
    }
}

File: ignored-test/csharp-xunit/InternalLoggerTests_Trace_ShouldThrowStackOverFlowExceptionWhenUsingNLogTraceListener.cs
using Xunit;

namespace None
{
    public class InternalLoggerTests_Trace
    {
        [Fact(Skip = "")]
        public void ShouldThrowStackOverFlowExceptionWhenUsingNLogTraceListener()
        {
            {
                SetupTestConfiguration<NLogTraceListener>(LogLevel.Trace, true, null);
                Assert.Throws<StackOverflowException>(() => Trace.WriteLine("StackOverFlowException"));
            }
        }
    }
}
